spec/lib/ecco/client_spec.rb

```diff
@@ -0,0 +1,71 @@
+require "spec_helper"
+
+describe Ecco::Client do
+  let(:client) do
+    described_class.new(
+      username: DatabaseHelper::USER,
+      password: DatabaseHelper::PASS,
+    )
+  end
+
+  describe "#get_server_id" do
+    let(:server_id) { 1337 }
+
+    before  { client.set_server_id(server_id) }
+    subject { client.get_server_id }
+
+    it { is_expected.to eq(server_id) }
+  end
+
+  describe "#is_keep_alive" do
+    subject { client.is_keep_alive }
+
+    context "when truthy" do
```

Comment:
Shouldn't this say "when true" since we're not setting nor checking for truthyness?